resources/lang/en.js

```diff
@@ -54,27 +54,27 @@ class CactbotLanguageEn extends CactbotLanguage {
     });
 
     this.kEffect = Object.freeze({
-      BluntResistDown: 'Blunt Resistance Down',
-      VerstoneReady: 'Verstone Ready',
-      VerfireReady: 'Verfire Ready',
-      Impactful: 'Impactful',
-      FurtherRuin: 'Further Ruin',
-      Aetherflow: 'Aetherflow',
-      WellFed: 'Well Fed',
-      OpoOpoForm: 'Opo-Opo Form',
-      RaptorForm: 'Raptor Form',
-      CoeurlForm: 'Coeurl Form',
-      PerfectBalance: 'Perfect Balance',
-      Medicated: 'Medicated',
-      BattleLitany: 'Battle Litany',
-      Embolden: 'Embolden',
-      Balance: 'The Balance',
-      Hypercharge: 'Hypercharge',
-      LeftEye: 'Left Eye',
-      RightEye: 'Right Eye',
-      Brotherhood: 'Brotherhood',
-      Devotion: 'Devotion',
-      FoeRequiem: 'Foe Requiem',
+      BluntResistDown: 'Blunt Resistance Down', // 0x23d, 0x335, 0x3a3
```

Comment:
I wasn't sure at first what these hex codes meant, but I looked through logs and I think I understand more of the log format.  I am very curious about this!

It seems like pairs of `:.*0F:(effectId):` designate the effect id.  Where did all of these values come from? Did you investigate these yourself or is there a list somewhere?

Some of these values, I see in logs, but others I do not.  For example, Foe Requiem up I see `[15:57:43.750] 15:1067BEA6:Azel Nox:73:Foe Requiem:1067BEA6:Azel Nox:90030F:8B:` but I do not see any :8C: for Foe Requiem lost.  Similarly, Left Eye and Right Eye are not abilities but only effects, and so I don't know where the values 0x4a0 and 0x49f come from.

Can you help me understand more? I'd love to know.  :)

Reply:
This is internal effect ID and used very widely. also skills, zones has unique ID like this.

`0x8C` is [for enemies](http://xivdb.com/status/140). (ps: XIVDB uses the decimal ID to build its URL)

Reply:
Don't miss out that we can access *unparsed log* like saved to a .log file, like [OverlayPlugin and SpecialSpellTimer do](https://github.com/hibiyasleep/OverlayPlugin/blob/master/OverlayPlugin.Core/Overlays/LogParseOverlay.cs#L29).

Reply:
Oh, I see! Thanks for the explanation.  By not seeing `0x8C`, I meant that I didn't see it anywhere in the log, applied to either players or enemies.

I'll have to investigate the unparsed log, thanks for the pointer!